CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+# v0.36.0
+
+## Push
+
+* Updated the default server host for the push service to match the production server.
```

Comment:
We don't edit `CHANGELOG.md` directly anymore, do you mind pushing these changes to `CHANGES_UNRELEASED.md`?

Reply:
Oops sorry, didn't know!